File: peek_plugin_diagram/_private/tuples/StringCapToggleActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_plugin_diagram._private.PluginNames import diagramTuplePrefix


@addTupleType
class StringCapToggleActionTuple(TupleActionABC):
    __tupleType__ = diagramTuplePrefix + "StringCapToggleActionTuple"

    stringIntId = TupleField()

File: peek_plugin_diagram/plugin-module/_private/tuples/StringCapToggleActionTuple.ts
                                       import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {diagramTuplePrefix} from "../PluginNames";

@addTupleType
export class StringCapToggleActionTuple extends TupleActionABC {
    static readonly tupleName = diagramTuplePrefix + "StringCapToggleActionTuple";

    stringIntId: number;

    constructor() {
        super(StringCapToggleActionTuple.tupleName)
    }
}